.gitignore

```diff
@@ -51,6 +51,7 @@ deployment/postgres-init-db/sql/**/*.sql
 
 # Ignore Gradle build output directory
 build
+bin
```

Comment:
Which of our Gradle build processes produces a `bin` directory that isn't inside an already-ignored directory?

Reply:
Oops, looks like these folders aren't actually created by Gradle, but instead the [jdtls](https://github.com/eclipse/eclipse.jdt.ls) build process if it can't properly interface with Gradle. If no other developers experience this, I can move this to my global `.gitignore` instead.

Reply:
Yeah, you may wanna double-check in the Slack Channel, but I don't _think_ anyone else is seeing this right now.